sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateEcKeyOptions.java

```diff
@@ -106,27 +106,27 @@ public EcKeyCreateOptions setTags(Map<String, String> tags) {
      * @param enabled The enabled value to set
      * @return the EcKeyCreateOptions object itself.
      */
-    public KeyCreateOptions setEnabled(Boolean enabled) {
+    public CreateKeyOptions setEnabled(Boolean enabled) {
```

Comment:
You're returning the wrong type here - you should return `CreateEcKeyOptions`

Reply:
Good catch, corrected.